client/src/Thumbnail.test.js

```diff
@@ -28,21 +47,21 @@ describe("Thumbnail", () => {
         const wrapper = mount(<Thumbnail {...props} />);
 
         expect(props.scrollTo).toHaveBeenCalled();
-        expect(props.getJobImageUrl).toHaveBeenCalledWith(props.number, "thumb");
+        expect(paginatorValues.action.getJobImageUrl).toHaveBeenCalledWith(paginatorValues.state.order[0], "thumb");
```

Comment:
Should you use props.number in place of the hard-coded 0 here? Should we add another test to cover a situation where there are multiple images? I wonder if testing the "one image only" edge case may not be sufficient on its own.

Reply:
(There are also some more hard-coded 0's below that might be best replaced with props.number).

Reply:
made change. added 3 objects.